src/drivers/rc_epgsql.erl

```diff
@@ -0,0 +1,55 @@
+-module (rc_epgsql).
+
+-behaviour (gen_server).
+
+-export ([start_link/1]).
+-export ([init/1, handle_info/2, handle_cast/2, handle_call/3, terminate/2]).
+
+start_link(Args) ->
+  gen_server:start_link({local, ?MODULE}, ?MODULE, Args, []).
+
+init(Args) ->
+  self() ! connect,
+  process_flag(trap_exit, true),
+  {ok, #{retries => 0, connection_args => Args}}.
+
+handle_info(connect, #{connection_args := ConnArgs} = State) ->
+  try
+    Host     = maps:get(host, ConnArgs, "localhost"),
```

Comment:
instead of having these maps:get with sensible defaults in handle_info (and potentially every time you need to access one of them), build the correct state once in init. That way you're sure that you'll always all required keys in the state after init.

To do that first constructing, maps:merge is probably more concise than a bunch of maps:get.